test/unit/mocks.js

```diff
@@ -1,13 +1,49 @@
 define(["ObjectMakr"], function (ObjectMakrModule) {
     var ObjectMakr = ObjectMakrModule.ObjectMakr;
+    var expect = require("chai").expect;
 
     var mocks = {
         /**
-         * 
+         * Returns an instance of ObjectMakr.
          */
         mockObjectMakr: function (settings) {
-            return new ObjectMakr(unitsize)
-        }
+            return new ObjectMakr(settings = {
```

Comment:
Default parameters are done with `||`, not `=`. The current way you're setting `settings` to the object literal regardless of whether a value was passed in.